hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/AbstractClientInvocationService.java

```diff
@@ -117,9 +117,13 @@ protected void send(ClientInvocation invocation, ClientConnection connection) th
         if (isShutdown) {
             throw new HazelcastClientNotActiveException("Client is shut down");
         }
+        ClientMessage clientMessage = invocation.getClientMessage();
+        if (clientMessage.isAcquiresResource() && connectionManager.getOwnerConnection() == null) {
```

Comment:
I would name this to hasAquiredResource.

Reply:
`acquiresResource` sounds better to me too. I am changing it.